drools-benchmarks/src/main/java/org/drools/benchmarks/turtle/runtime/KBaseCreationFromDTable1Benchmark.java

```diff
@@ -0,0 +1,54 @@
+package org.drools.benchmarks.turtle.runtime;
+
+import org.drools.benchmarks.common.util.RuntimeUtil;
+import org.drools.benchmarks.model.Account;
+import org.kie.api.runtime.KieSession;
+import org.kie.api.runtime.conf.ClockTypeOption;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Setup;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class KBaseCreationFromDTable1Benchmark extends AbstractSimpleRuntimeBenchmark {
```

Comment:
Please tweak the warmup and measurement iterations here - the ones in the parent are just the default ones. Usually each benchmark needs its specific number of warmup and measurement iterations.

Reply:
@Warmup(iterations = 60, time = 50)
Warmup Iteration 1: 41.254 ms/op
After 40 iterations it’s stable on ~5 ms/op

@Measurement(iterations = 20, time = 10)
No more time than 10 ms (sometimes it reaches 7.539 ms/op)